sklearn/dummy.py

```diff
@@ -90,6 +90,9 @@ class DummyClassifier(MultiOutputMixin, ClassifierMixin, BaseEstimator):
     array([1, 1, 1, 1])
     >>> dummy_clf.score(X, y)
     0.75
+    ..vershionchanged:: v0.19
```

Comment:
typo. also needs newline I think?

Reply:
Editing this for typos and formatting. Will resend pull request.